src/core/json/path.cc

```diff
@@ -475,10 +476,12 @@ nonstd::expected<json::Path, string> ParsePath(string_view path) {
   return driver.TakePath();
 }
 
-void MutatePath(const Path& path, MutateCallback callback, JsonType* json) {
+unsigned MutatePath(const Path& path, MutateCallback callback, JsonType* json) {
   if (path.empty())
-    return;
-  Dfs().Mutate(path, callback, json);
+    return 0;
+  Dfs dfs;
+  dfs.Mutate(path, callback, json);
```

Comment:
maybe we can create static method Mutate that return Dfs object and we don't need to create the object before

Reply:
I do not see a problem with this code.